src/app/containers/ArticleTimestamp/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,133 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ArticleTimestamp should handle incomplete data 1`] = `null`;
+
+exports[`ArticleTimestamp should not return values for invalid timestamps 1`] = `
```

Comment:
Would we expect this to render an empty `<div>` as a grid item? I'd assume if we aren't rendering a timestamp value we shouldn't render the `<div>`

Reply:
Added a check to stop empty <div> being rendered.